web/modules/custom/usagov_wizard/usagov_wizard.module

```diff
@@ -408,22 +408,22 @@ function usagov_wizard_get_term_breadcrumb(Term $rootTerm): array {
   if (isset($rootTerm->field_wizard_breadcrumbs)) {
     $crumbs = $rootTerm->field_wizard_breadcrumbs->getValue();
     $result = [];
+
     foreach ($crumbs as $crumb) {
       if (isset($crumb['uri'])) {
-        $crumb['url'] = Url::fromUri($crumb['uri'])->toString();
-        // Sometimes Drupal doesn't prepend Spanish path prefix.
         // Sometimes this crumb's uri is "route:<nolink>" and the url is an empty string;
```

Comment:
Please remove this comment too! 
(I'm just getting into the tome-running and ppr-reviewing, so you'll probably get to that before I'm ready to approve.)

Reply:
Remove this comment?

```
// Sometimes this crumb's uri is "route:<nolink>" and the url is an empty string;
// in that case we should leave it empty (current page, not to be linked).
```

Reply:
Yes

Reply:
removed